e2e-test/cypress/integration/preselect_language_spec.js

```diff
@@ -0,0 +1,27 @@
+describe("Checking if language ist still selected after Logout", function () {
+
+  let languageArray = ["en-gb", "fr", "pt", "de"];
```

Comment:
use const here; Put the language shortcuts into helper module; Don't put the var type into var name.